mumd/src/state.rs

```diff
@@ -687,14 +687,26 @@ pub fn handle_command(
 
             for target in targets {
                 match target {
+                    MessageTarget::CurrentChannel { recursive } => {
+                        let channel_id = match state.server().unwrap().current_channel() {
+                            Some(channel)=> channel.0,
+                            None => return now!(Err(Error::NotConnectedToChannel)),
+                        };
+
+                        if recursive {
+                            msg.mut_tree_id()
+                        } else {
+                            msg.mut_channel_id()
+                        }
+                        .push(channel_id);
```

Comment:
Having this statement hang free looks a bit weird. Maybe store the reference in a temporary variable and then push to it?